tests/profiling/exporter/test_pprof.py

```diff
@@ -652,15 +653,15 @@ def test_sequence():
 
 
 def test_string_table():
```

Comment:
You should remove this or move it to `ddtrace.internal.encoding` tests if you don't have it already.
No need to duplicate the test!